posts/2015-07-25-new-javascript-generators.md

```diff
@@ -0,0 +1,120 @@
+---
+title: New JavaScript Generators
+author: Freezeboy
+date: 2015-07-25 00:00
+toc: true
+---
+
+# Summary
+
+With the upcoming Servant version, it will be way easier to build a javascript
+connector to your webservice. The chosen API is *as usual with Servant*
+putting the magic away from your eyes to let you concentrate on what you are
+working on in most situations.
+
+Extensibility is still available, again *as usual with Servant*, each part of
+this library is kind of a plugin, and it is future-proof: If you want more, just
+get it, it's possible.
+
+# The Problem
+
+Until Servant 0.4 branch, we had a javascript generator using exclusively
+JQuery, but depending on the framework you might want to use in your project,
+it can be overkill to fetch [JQuery](http://jquery.org) library and for example
```

Comment:
maybe "overkill to fetch JQuery, or else you are using a library that prefers doing things differently, such as AngularJS".